WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java

```diff
@@ -660,10 +660,6 @@ public GravatarLoadFinished(boolean success) {
 
     public void onEventMainThread(GravatarLoadFinished event) {
         showGravatarProgressBar(false);
-
-        if (!event.success) {
```

Comment:
Hey @nbradbury , instead of removing the toast, may I suggest guarding it with a `mIsUpdatingGravatar` check as well? Originally, the toast was added to inform the user of errors on reloading the Gravatar after updating it so, it should suffice to have it only in that case.

As a headsup, and if you choose to add the check, notice that `mIsUpdatingGravatar` is modified by `showGravatarProgressBar()` so you might want to re-arrange the code lines order here. Thanks!